roles/nzbhydra2/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Create NZBHydra2 Directories
+  file:
+    path: "{{ item }}"
+    state: directory
+  with_items:
+    - "{{ nzbhydra2_data_directory }}"
+
+- name: NZBHydra2
+  docker_container:
+    name: nzbhydra2
+    image: linuxserver/nzbhydra2
+    pull: true
+    volumes:
+      - "{{ nzbhydra2_data_directory }}:/config:rw"
+    ports:
+      - "5076:5076"
+    env:
+      TZ: "{{ ansible_nas_timezone }}"
+      PUID: "{{ nzbhydra2_user_id }}"
+      PGID: "{{ nzbhydra2_group_id }}"
+    restart_policy: unless-stopped
+    memory: 1g
```

Comment:
Can you make this configurable please, similar to in other roles?

Reply:
I made both the changes. The first one I just missed but this one was because the role I chose to copy still had hardcoded memory. Would it be useful to make a PR to unify the memory config on all of the roles?